src/scale.ts

```diff
@@ -611,7 +611,7 @@ export interface Scale<ES extends ExprRef | SignalRef = ExprRef | SignalRef> {
    *
    * For the full list of supported schemes, please refer to the [Vega Scheme](https://vega.github.io/vega/docs/schemes/#reference) reference.
```

Comment:
Should we advertise this capability in the docs and/or a docstring? 

https://vega.github.io/vega-lite/docs/scale.html#2-setting-the-range-property-to-an-array-of-valid-css-color-strings seems close but it targets `scale.range` rather advertising allowing `scheme` to be immediately set as an array. I'm assuming this arrays  is a faster shorthand for the same behavior.

Reply:
Good point. I think we should revert this change since there ideally aren't multiple ways to do the same thing.

Reply:
I sent https://github.com/vega/vega-lite/pull/9266